manifests/params.pp

```diff
@@ -106,7 +106,11 @@
         default => 'mod_authz_ldap',
       },
       'authnz_pam'            => 'mod_authnz_pam',
-      'fastcgi'               => 'mod_fastcgi',
+      'fastcgi'               => $::apache::version::distrelease ? {
```

Comment:
Wouldn't this produce undefined results on other distros like Debian?

Reply:
If you look here https://github.com/tphoney/puppetlabs-apache/blob/c891950899a35a4af79fb23ce123f29d6da3639c/manifests/params.pp#L62-L110 It is for centos only @ekohl

Reply:
You're right, my bad. It's not an early morning but it does feel like it ...

Reply:
its raining and miserable here, it isnt good. Thanks for looking over the PR. Much appreciated.